Source/Particles/Collision/BinaryCollision/Coulomb/UpdateMomentumPerezElastic.H

```diff
@@ -22,6 +22,8 @@
  *        @param[in] L is the Coulomb log. A fixed L will be used if L > 0,
  *        otherwise L will be calculated based on the algorithm.
  *        @param[in] n12 = max(w1,w2)*min(N1,N2)/dV is the effective density used for s12
+ *        @param[in] sigma_max is the maximum cross section based on mfp = atomic spacing
+ *        used for the normalized scattering length s12
```

Comment:
Could you add a reference (e.g. pointing to a specific section of the Perez paper), so that future developers that read this understand where this comes from?